router/router.gno

```diff
@@ -139,76 +184,58 @@ func SwapRoute(
 	return amountIn, amountOut
 }
 
-func validateInput(amountSpecified *i256.Int, swapType string, routes, quotes []string) error {
-	if amountSpecified.IsZero() || amountSpecified.IsNeg() {
-		return ufmt.Errorf(
-			"%s: amountSpecified(%s), must be positive",
-			errInvalidInput.Error(), amountSpecified.ToString(),
-		)
-	}
-
-	if len(routes) < 1 || len(routes) > 7 {
-		return ufmt.Errorf(
-			"%s: route length must be 1~7. got %d",
-			errInvalidInput.Error(), len(routes),
-		)
+func DrySwapRoute(
```

Comment:
Where is `DrySwapRoute` being used? Is this necessary?

Reply:
Like `pool.DrySwap`, it seems to be used for simulation purposes.

Reply:
@onlyhyde @r3v4s @notJoon can we check if these dry functions are being used in the transaction process? if so I want to understand how exactly they're used. I see `DrySwap`, `DrySwapRoute`, and `DryMint`, but add more if there are.

Reply:
Delete `DrySwapRoute` if not being used.

Reply:
removed https://github.com/gnoswap-labs/gnoswap/pull/431/commits/ae8bedebcd7b883617adbc5c4431062e8284eda9